op-node/rollup/driver/driver.go

```diff
@@ -249,24 +247,20 @@ func NewDriver(
 
 	driverEmitter := sys.Register("driver", nil)
 	driver := &Driver{
-		statusTracker:    statusTracker,
-		SyncDeriver:      syncDeriver,
-		sched:            schedDeriv,
-		emitter:          driverEmitter,
-		drain:            drain.Drain,
-		stateReq:         make(chan chan struct{}),
-		forceReset:       make(chan chan struct{}, 10),
-		driverConfig:     driverCfg,
-		driverCtx:        driverCtx,
-		driverCancel:     driverCancel,
-		log:              log,
-		sequencer:        sequencer,
-		metrics:          metrics,
-		l1HeadSig:        make(chan eth.L1BlockRef, 10),
-		l1SafeSig:        make(chan eth.L1BlockRef, 10),
-		l1FinalizedSig:   make(chan eth.L1BlockRef, 10),
-		unsafeL2Payloads: make(chan *eth.ExecutionPayloadEnvelope, 10),
-		altSync:          altSync,
+		statusTracker: statusTracker,
+		SyncDeriver:   syncDeriver,
+		sched:         schedDeriv,
+		emitter:       driverEmitter,
+		drain:         drain,
+		stateReq:      make(chan chan struct{}),
+		forceReset:    make(chan chan struct{}, 10),
```

Comment:
is this eventually also something that you want to hook up to the events system?

Reply:
yes, probably better to make this part of one of the following op-node driver PRs though.